product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx

```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
 #export QUARKUS_LOG_CATEGORY__COM_ENTERPRISEDB__LEVEL=DEBUG
 ```
 
-## Running the reader
-
-```shell
-cd /opt/cdcreader
-./run-cdcreader.sh
-```
-
 ### Parameters explanation
 
 #### DBZ_ID
-ID to identify the reader. Users can define the ID they think will be useful with below constraints:
 
-1. ID length should lower or equal to 255
-2. ID can only contain lowercase characters, uppercase characters, numbers, underscore(_) and hyphen(-)
-3. ID should be unique, two reader instances should not use the same ID
+This is the name you assign to a migration to identify it. 
+Take into account the following limitations: 
+
+- The ID cannot be longer than 255 characters.
+- The ID can contain lowercase characters, uppercase characters, numbers, underscores(_) and hyphens(-)
+- The ID must be unique; two reader instances should not have the same ID.
 
 #### RW_SERVICE_HOST
-URL of the transporter-rw-service is always https://transporter-rw-service.biganimal.com
+
+Uncomment the line. The URL of the `transporter-rw-service` is always https://transporter-rw-service.biganimal.com.
```

Comment:
Why do we need to add "Uncomment the line." here

Reply:
You are right, we are not suggesting to uncomment all other lines of the reader when configuring them, so we should not explicitly call it out here. What about this? 

```suggestion
Specifies the URL of the service that will host the migration. `transporter-rw-service` is always https://transporter-rw-service.biganimal.com.
```